src/utils/shell_compatibility.ts

```diff
@@ -22,6 +22,22 @@ function metaSupportsUnredirectForDisplay() {
   );
 }
 
+// Meta.Cursor.POINTING_HAND was renamed to Meta.Cursor.POINTER in GNOME 48 (Meta 16)
+
+interface NewMetaCursor {
+  POINTER: Meta.Cursor | null | undefined;
+}
+
+export const MetaCursorPointer: Meta.Cursor = (() => {
+  const pointer = (Meta.Cursor as unknown as NewMetaCursor).POINTER;
+
+  if (pointer !== undefined && pointer !== null) {
```

Comment:
The value should never be null

Reply:
I never checked whether the C to JS bindings (since Meta is written in C) return undefined or null in such cases for enums, but it is also not documented, so undefined behavior, I like to guard it strongly, this is only executed once, so it's fine